package.json

```diff
@@ -29,18 +29,8 @@
     "license": "MIT",
     "icon": "icon.png",
     "activationEvents": [
-        "onCommand:vscode-pets.start",
```

Comment:
Can be remove as VS Code generates them automatically from the package.json contribution declarations